perception/image_projection_based_fusion/src/roi_cluster_fusion/node.cpp

```diff
@@ -177,7 +177,7 @@ void RoiClusterFusionNode::fuseOnSingleImage(
       if (use_iou_y_) {
         iou_y = calcIoUY(cluster_map.second, feature_obj.feature.roi);
       }
-      if (max_iou < iou + iou_x + iou_y) {
+      if (max_iou < iou + iou_x + iou_y && is_inside(feature_obj.feature.roi, cluster_map.second)) {
```

Comment:
I would like to have the option to choose.

Reply:
updated in https://github.com/autowarefoundation/autoware.universe/pull/4147/commits/9fa0df5259dc534859c4b3beda72ab09640a1acd

Reply:
LGTM